src/System.Security.Cryptography.X509Certificates/tests/PublicKeyTests.cs

```diff
@@ -151,6 +166,35 @@ public static void TestKey_RSA384_ValidatesSignature()
             }
         }
 
+        [Theory, MemberData(nameof(BrainpoolCurves))]
+        public static void TestKey_ECDsabrainpool_PublicKey(byte[] curveData, byte[] notUsed)
```

Comment:
The BrainpoolCurves method also returns a signature for the curveData but that is not used by this test method (and the test infrastructure doesn't support optional params). It is used by another test method, however. I could split the data methods into two to avoid the unused parameter, but thought this was straightforward enough.